docs/platform/howto/restrict-access.md

```diff
@@ -5,40 +5,42 @@ sidebar_label: Restrict access
 
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-By default, Aiven services are publicly accessible, but you can restrict access to your service to a single IP, an address block, or any combination of both.
+Restrict access to your Aiven-managed service to a single IP, an address block, or any combination of both.
 
-:::important
-IP filters apply to inbound traffic coming from specified IP addresses / network ranges to
-Aiven service ports.
+By default, Aiven services are publicly accessible. To restrict access to Aiven services,
+you can use the IP filtering capability, which allows you to filter traffic incoming to
+your services by specifying allowed IP addresses / network ranges.
+
+note:::
+If your service is within a VPC, the VPC configuration filters incoming traffic before the
+IP filter is applied.
 :::
 
+By default, the IP filter is set `0.0.0.0/0`, which allows all inbound connections. If you
+remove `0.0.0.0/0`, your service loses all connectivity, including private network addresses.
+
 :::tip
 To access a non-publicly-accessible service from another service, use a
 [service integration](/docs/platform/concepts/service-integration).
 :::
 
 ## Restrict access
 
-1. Log in to [Aiven Console](https://console.aiven.io).
-
-1. On the **Services** page, select the service to restrict.
-
-1. On the **Overview** page of your service, select **Service
-    settings** from the sidebar.
-
-1. On the **Service settings** page, in the **Cloud and
+1. Log in to [Aiven Console](https://console.aiven.io), and select the service to restrict
+   access to.
+1. On the <ConsoleLabel name="overview"/> page of your service, select
+   <ConsoleLabel name="service settings"/>.
+1. On the <ConsoleLabel name="service settings"/> page, in the **Cloud and
    network** section, click <ConsoleLabel name="actions"/> > **Set public IP filters**.
-
-1. In the **Allowed inbound IP addresses** window, enter your address
-   or address block using the CIDR notation, and select the **+** icon
-   to add it to the list of the trusted IP addresses.
+1. In the **Allowed inbound IP addresses** window, click **Add IP address range**, and
+   enter your address or address block using the CIDR notation, for example `10.20.0.0/16`.
```

Comment:
If the customer adds a new IP range like this, and does not remove the default 0.0.0.0/0 (which allows everything), the service is still wide open to the world, and the change has no effect.

When adding the first subnet there, the default of 0.0.0.0/0 must be replaced with 10.20.0.0/16, then click "Save changes". If additional subnets beyond that need to be allowed, the "Add IP address range" button must be clicked.